bpdm-gate-api/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/api/model/IBaseBusinessPartnerGateDto.kt

```diff
@@ -28,6 +28,6 @@ interface IBaseBusinessPartnerGateDto : IBaseBusinessPartnerDto {
     @get:Schema(description = CommonDescription.externalId)
     val externalId: String
 
-    @get:Schema(name = "isOwner", description = "True if the sharing member declares itself as the owner of the business partner.")
+    @get:Schema(name = "isOwnCompanyData", description = "True if the sharing member declares itself as the owner of the business partner.")
```

Comment:
better to just remove the whole name override. No need to have a name override there. Springdoc should just take the name of the field instead